### DISCUSSION TITLE
colab

### DISCUSSION THREAD

**Discussion by alessiofumagalli** [pmgbergen/porepy#982] [Ideas]
Dear all,

if you are interested I attach (to be unpacked) a configuration file to use pp in colab.
[porepy_colab.sh.zip](https://github.com/pmgbergen/porepy/files/12764738/porepy_colab.sh.zip)

On colab upload the file and to trigger the installation simply run
```
!source porepy_colab.sh
```

Cheers.

**Comment by keileg** [comment]
Thanks. We may look into this at some point, though we may choose to go with a solution based on GH codespaces instead. TBC.
